common/app/model/pressedContent.scala

```diff
@@ -67,7 +68,8 @@ final case class CollectionConfig(
   excludeFromRss: Boolean,
   showTimestamps: Boolean,
   hideShowMore: Boolean,
-  displayHints: Option[DisplayHints]
+  displayHints: Option[DisplayHints],
+  platform: Option[CollectionPlatform] = None
```

Comment:
Why is it an `Option`?

Reply:
The platform field is optional here because it needs to read in existing pressed data from s3. A missing platform field means it's an Any platform collection.